dlib/CMakeLists.txt

```diff
@@ -601,6 +606,15 @@ if (NOT TARGET dlib)
          endif()
       endif()
 
+      include(cmake_utils/find_ffmpeg.cmake)
+      if (DLIB_USE_FFMPEG)
+         if (FFMPEG_FOUND)
+            set (dlib_needed_libraries ${dlib_needed_libraries} ${FFMPEG_LIBRARIES})
```

Comment:
@arrufat Isn't it ```${libavdevice_LDFLAGS} ${libavfilter_LDFLAGS} ${libavformat_LDFLAGS} ${libavcodec_LDFLAGS} ${libswresample_LDFLAGS} ${libswscale_LDFLAGS}  ${libavutil_LDFLAGS} ``` ??

Reply:
Actually, what i suggested is wrong. But, from [the docs](https://cmake.org/cmake/help/latest/module/FindPkgConfig.html), ```FFMPEG_LIBRARIES``` doesn't provide you with the full linker flags. I think that's why it wasn't working on my machine. I think we need to use ```FFMPEG_LDFLAGS``` somehow